metrologist/field_illum/models.py

```diff
@@ -1,3 +1,77 @@
+# -*- coding: utf-8 -*-
+"""User models."""
+import datetime as dt
+import logging
+
+from metrologist.database import (
+    Column,
+    PkModel,
+    db,
+    reference_col,
+    relationship,
+)
+
+log = logging.getLogger(__name__)
+
+import homogeneity_module as homo
+import common_module as cm
+
+"""
+sqlalchemy() imported as db (db = SQLAlchemy())
+when imported metrologist.database
+"""
+
+metadata = db.MetaData()
+
+"""
+Rk: missing: microscopy_info_table defined from microscopes folder
+as it's supposed to be part of "microscopes" dir rather than
+homogeneity. 
+"""
+
+
 class Homogeneity(PkModel):
+    """
+    Homogeneity class refers to homogeneity report generating
+    from a given image:
+        1. norm_intensity_data: np.array used to generate the normalized
+        intensity profile of the image. It's used to generate the corresponding
+        image.
+        2. max_region_table: dataframe enclosing information about the max
+        intensity area of the given image.
+        3. intensity_plot_data: np.array used to generate the intensity plot of
+        the mid horizontal, mid vertical and the two diagonal lines of the image.
+        4. profile_stat_table: dataframe showing the intensity values of 9
+        specific pixels and their ratio over the maximum intensity value of
+        the array.
+    """
+            
     __tablename__ = "homogeneities"
-    top_left = Column(db.BigInter())
+
```

Comment:
I think it's better if you directly define the columns as attributes of the `Homogeneity` class. If you want 3 tables, then there should be three `PkModel` classes